azure-kusto-data/tests/test_kusto_client.py

```diff
@@ -104,7 +104,7 @@ def test_sanity_query(self, mock_post):
             "xnumberAsText": text_type(""),
             "xtime": None,
             "xtextWithNulls": text_type(""),
-            "xdynamicWithNulls": None,
```

Comment:
why did you remove the null checks? (IIUC). can you test null dynamic values (not empty strings)?

Reply:
I am testing it at [test_dynamic](https://github.com/Azure/azure-kusto-python/blob/2aa575f1442c307ff5d7e4f021aa9d1760f1cb8b/azure-kusto-data/tests/test_kusto_client.py#L350) at test_kusto_client.py.